### PR TITLE
Chore/reorg no testing/137 - Subset PR

### PR DESCRIPTION
**Pull request by saraswatpuneet**
Problem
=======
The separation between DSNP spec and chain level implementation was not clear or easy to see.
Link to GitHub Issue: https://github.com/LibertyDSNP/spec/issues/137

Solution
========
What I/we did to solve this problem

with @wilwade 

Change summary:
---------------
* Tidy up Roadmap
* DNSP overview to include changelog related to version releases
*  ....

### CODE REVIEW

**Comment by saraswatpuneet** [comment]
@wilwade  using this PR has placeholder going to put more changes I deem useful, and we can later merge into your branch

**Review by shannonwells — APPROVED** [review]
🥳 🚢

File: pages/DSNP/Announcements.md [hunk 1]
```diff
@@ -4,16 +4,11 @@ name: Announcements
...
-# TODO
-
-I took out signatures from this level. Do we still need them for DSNP 1.0?
-
-
 # Announcements Overview
 
 Announcements are content or reference to content that communicate new user activity to the rest of the network.
 Announcements are associated with an [Identifier](/DSNP/Identifiers) which can be validated as creator of the announcement.
+Announcement validation starts with the [signature](/DSNP/Signatures) of the announcement.
```
Comment (wilwade) [inline pages/DSNP/Announcements.md:11]: Now that I read this line again I think we should revisit this line. I'm thinking re-word it and then add a section below about `## Announcement Validation`? Thoughts @saraswatpuneet ?

Comment (saraswatpuneet) [reply pages/DSNP/Announcements.md:11]: @wilwade  I agree, let me try it out  then we can discuss tomorrow/pair

File: pages/DSNP/Announcements.md [hunk 2]
```diff
@@ -4,18 +4,16 @@ name: Announcements
...
-
 # Announcements Overview
 
 Announcements are content or reference to content that communicate new user activity to the rest of the network.
 Announcements are associated with an [Identifier](/DSNP/Identifiers) which can be validated as creator of the announcement.
 Depending on the implementation Announcements may be published directly to the network, included in [Batch Publication Files](/DSNP/BatchPublications), or some combination of those two.
 
+## Announcement Validation
+
+[Signature](/DSNP/Signatures) is the first step in validating an announcement.
```
Comment (wilwade) [inline pages/DSNP/Announcements.md:15]: I think we will need to expand this. Perhaps something along these lines?

```markdown
There is no guarantee at time of creation that a given announcement will be from the `fromId` claimed in the announcement.
The reader of the announcement MUST perform a validation at read time to ensure authenticity.
Implementations MUST provide a way to validation what [signatures](/DSNP/Signatures) are associated with a given [identifier](/DSNP/Identifiers).
```

Hmm... I like how that is readable, but I wonder if we need something closer to how we do `Ordering Announcements`?

Thoughts @saraswatpuneet?

Comment (saraswatpuneet) [reply pages/DSNP/Announcements.md:15]: I like it better, let me use those lines

File: pages/DSNP/Announcements.md [hunk 3]
```diff
@@ -4,18 +4,18 @@ name: Announcements
...
 Announcements are content or reference to content that communicate new user activity to the rest of the network.
 Announcements are associated with an [Identifier](/DSNP/Identifiers) which can be validated as creator of the announcement.
 Depending on the implementation Announcements may be published directly to the network, included in [Batch Publication Files](/DSNP/BatchPublications), or some combination of those two.
 
+## Announcement Validation
+
+There is no guarantee that, at time of creation, a given announcement will be from the `fromId` claimed in the announcement.
+The reader MUST perform a validation of the announcement at read time to ensure authenticity.
+Implementations MUST provide a way to validate what [signatures](/DSNP/Signatures) are associated with a given [identifier](/DSNP/Identifiers).
+
```
Comment (saraswatpuneet) [inline pages/DSNP/Announcements.md:18]: @wilwade  how does that, moved around to read better

Comment (wilwade) [reply pages/DSNP/Announcements.md:18]: Nice! Let's get this PR merged and we can make a new one for additional changes.

**Review by wilwade — APPROVED** [review]

**Pull request merged**